scripts/package-harvester-os

```diff
@@ -151,9 +151,21 @@ spec:
 EOF
 fi
 
-# Collect image lists to a tarball "image-lists.tar.gz"
+# Collect image lists
 OUTPUT_DIR="$TOP_DIR/dist/artifacts/image-lists"
 mkdir -p $OUTPUT_DIR
 find $IMAGES_LISTS_DIR -name "*.txt" -exec cp {} $OUTPUT_DIR \;
 find $RANCHERD_IMAGES_DIR -name "*.txt" -exec cp {} $OUTPUT_DIR \;
+
+# Write all images into one file for user convenience
+IMAGE_ALL=$TOP_DIR/dist/artifacts/image-all.txt
```

Comment:
Just a style discussion,
Could we use `harvester-image-lists`?  That might be more readable?
WDUT?

Reply:
The name is copied from `image-lists.tar.gz` lazily.

Some candidates: `harvester-images-set.txt`, `harvester-image-lists.txt`, `harvester-images-all.txt`, `image-all.txt`...

@guangbochen @Vicente-Cheng which one do you prefer?

Reply:
`harvester-images-list.txt` looks good to me, thanks.

Reply:
The `harvester-images-list.txt` is used now. thanks.